src/main/java/application/rest/v1/actions/FunctionResolver.java

```diff
@@ -69,13 +73,22 @@ public String resolve(ResolutionContext context, String suffix) throws PatternEx
                 }
                 // One of the function parameters couldn't be resolved.
                 else {
+                    if (Logger.isExitEnabled()) {
+                        Logger.log(FunctionResolver.class.getName(), "resolve", Logger.LogType.EXIT, "Return null.");
+                    }
                     return null;
                 }
             }
             
             // Invoke the function.
+            if (Logger.isExitEnabled()) {
```

Comment:
This is a case where we could revise the code a bit to capture the return value in the exit log:

final String result = function.invoke(...);
if (...)
    Logger.log(..., result);
return result;

Reply:
Updated.